frontend/package.json

```
@@ -7,58 +7,58 @@
   "dependencies": {
     "@aspida/fetch": "^1.14.0",
     "@aspida/node-fetch": "^1.14.0",
-    "@faker-js/faker": "^7.5.0",
-    "@headlessui/react": "^1.6.6",
-    "@iconify/react": "^3.2.2",
+    "@faker-js/faker": "^8.4.1",
+    "@headlessui/react": "^1.7.18",
+    "@iconify/react": "^4.1.1",
     "ajv": "^8.12.0",
     "ajv-formats": "^2.1.1",
     "aspida": "^1.14.0",
-    "clsx": "^2.0.0",
+    "clsx": "^2.1.0",
     "fast-glob": "^3.3.2",
-    "image-size": "^1.0.2",
-    "jotai": "^2.5.1",
-    "msw": "^2.0.6",
-    "next": "^14.0.2",
+    "image-size": "^1.1.1",
+    "jotai": "^2.7.0",
+    "msw": "^2.2.2",
+    "next": "^14.1.1",
     "next-mdx-remote": "^4.4.1",
-    "node-fetch": "^2.7.0",
+    "node-fetch": "^3.3.2",
```

https://github.com/node-fetch/node-fetch?tab=readme-ov-file#commonjs v3系はCommonJSに対応していない

```suggestion
    "node-fetch": "^2.7.0",
```